docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc

```diff
@@ -60,21 +65,21 @@ export PULL_OPENSTACK_CONFIGURATION_NOVA_COMPUTE_HOSTNAMES=$(oc run mariadb-clie
 echo "$PULL_OPENSTACK_CONFIGURATION_NOVA_COMPUTE_HOSTNAMES"
 ----
 
-. Get the list of mapped the {compute_service} cells:
+. Get the list of the mapped {compute_service} cells:
 +
 ----
 export PULL_OPENSTACK_CONFIGURATION_NOVAMANAGE_CELL_MAPPINGS=$($CONTROLLER1_SSH sudo podman exec -it nova_api nova-manage cell_v2 list_cells)
 echo "$PULL_OPENSTACK_CONFIGURATION_NOVAMANAGE_CELL_MAPPINGS"
 ----
 +
-After the source control plane services shutdown, if either of the exported
-values lost, it could be no longer evaluated again. Preserving the exported
-values in an environment file should protect you from such a situation.
+[IMPORTANT]
+After the {OpenStackShort} control plane services are shut down, if any of the exported
+values are lost, re-running the command fails because the values cannot be evaluated again. To avoid data loss, preserve the exported values in an environment file before shutting down the control plane services.
```

Comment:
Will the user understand "the values cannot be evaluated again"?

Reply:
I have elaborated the meaning in the comment above: "the values cannot be evaluated again" - if we re-run the commands, they will fail as there are no longer control plane services running on the source cloud to get the data from (and those cannot be started back).

Feel free to rephrase, with preserving the original intent

Reply:
Thanks @bogdando . When you say, re-run the commands, do you mean the "export" commands specifically, or any commands in this procedure?

Reply:
None of the commands in this chapter would work after its relates services stopped on the source cloud